### ISSUE TITLE
Fatal Python error: PyThreadState_Get: no current thread

### ISSUE THREAD

**Issue by christianbrodbeck** [conda-forge/vtk-feedstock#75]
Issue:

I currently get the error below from a direct installation of `vtk` from conda-forge. This did not use to happen, so I'm wondering whether it's related to a system update or update in some other package. It happens on both Macs I'm using (10.14.3 (18D42) and 10.14.2 (18C54)). Can others replicate?

```bash
$ conda create -n vtk_test -c conda-forge vtk 
...
$ conda activate vtk_test
$ python -c "import vtk"
Fatal Python error: PyThreadState_Get: no current thread

Abort trap: 6

```
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/christian/anaconda3/envs/vtk_test:
#
# Name                    Version                   Build  Channel
ca-certificates           2018.11.29           ha4d7672_0    conda-forge
certifi                   2018.11.29            py36_1000    conda-forge
curl                      7.63.0            heae2a1f_1000    conda-forge
expat                     2.2.5             h0a44026_1002    conda-forge
freetype                  2.9.1             h597ad8a_1005    conda-forge
future                    0.17.1                py36_1000    conda-forge
hdf4                      4.2.13            hf3c6af0_1002    conda-forge
hdf5                      1.10.1                        2    conda-forge
icu                       58.2              h0a44026_1000    conda-forge
jpeg                      9c                h1de35cc_1001    conda-forge
jsoncpp                   1.8.1                h2d50403_1    conda-forge
krb5                      1.16.3            h24a3359_1000    conda-forge
libcurl                   7.63.0            h76de61e_1000    conda-forge
libcxx                    7.0.0                h2d50403_2    conda-forge
libedit                   3.1.20170329      hcfe32e1_1001    conda-forge
libffi                    3.2.1             h0a44026_1005    conda-forge
libgfortran               3.0.0                      1001    conda-forge
libiconv                  1.15              h1de35cc_1004    conda-forge
libnetcdf                 4.6.1                         2    conda-forge
libogg                    1.3.2             h1de35cc_1001    conda-forge
libpng                    1.6.36            ha441bb4_1000    conda-forge
libssh2                   1.8.0             hf30b1f0_1003    conda-forge
libtheora                 1.1.1             h1de35cc_1001    conda-forge
libtiff                   4.0.10            h79f4b77_1001    conda-forge
libvorbis                 1.3.5             h1de35cc_1001    conda-forge
libxml2                   2.9.8             hf14e9c8_1005    conda-forge
llvm-meta                 7.0.0                         0    conda-forge
lz4-c                     1.8.1.2                       0    conda-forge
ncurses                   6.1               h0a44026_1002    conda-forge
openssl                   1.0.2p            h1de35cc_1002    conda-forge
pip                       19.0.1                   py36_0    conda-forge
python                    3.6.7             h4a56312_1001    conda-forge
readline                  7.0               hcfe32e1_1001    conda-forge
setuptools                40.7.1                   py36_0    conda-forge
sqlite                    3.26.0            h1765d9f_1000    conda-forge
tbb                       2019.3            h04f5b5a_1000    conda-forge
tk                        8.6.9             ha441bb4_1000    conda-forge
vtk                       8.1.1           py36h8649a91_204    conda-forge
wheel                     0.32.3                   py36_0    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
zlib                      1.2.11            h1de35cc_1004    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : vtk_test
    active env location : /Users/christian/anaconda3/envs/vtk_test
            shell level : 2
       user config file : /Users/christian/.condarc
 populated config files : /Users/christian/.condarc
          conda version : 4.6.1
    conda-build version : 3.17.7
         python version : 3.7.1.final.0
       base environment : /Users/christian/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/christian/anaconda3/pkgs
                          /Users/christian/.conda/pkgs
       envs directories : /Users/christian/anaconda3/envs
                          /Users/christian/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.1 requests/2.19.1 CPython/3.7.1 Darwin/18.2.0 OSX/10.14.3
                UID:GID : 501:20
             netrc file : None
           offline mode : False

```
</details>

**Comment by looooo** [comment]
can you try latest available vtk (8.2.0)?

**Comment by christianbrodbeck** [comment]
No more error, the issue seems to be fixed.
